tests/test_text_onnx_embeddings.py

```diff
@@ -16,6 +16,7 @@
     "sentence-transformers/all-MiniLM-L6-v2": np.array([0.0259, 0.0058, 0.0114, 0.0380, -0.0233]),
     "sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2": np.array([0.0094, 0.0184, 0.0328, 0.0072, -0.0351]),
     "intfloat/multilingual-e5-large": np.array([0.0098, 0.0045, 0.0066, -0.0354, 0.0070]),
+    "intfloat/multilingual-e5-small": np.array([0.04931236, 0.02415175, -0.0384715, -0.08884481, 0.08710264]),
```

Comment:
This is not the vector you get after normalization: https://colab.research.google.com/drive/1tNdV3DsiwsJzu2AXnUnoeF5av1Hp8HF1?usp=sharing

This looks like you've copied the vector from the ONNX test case and pasted here, please don't do that!

Reply:
This is NTO Resolved. The test vector needs to come from a source implementation which we can trust. For instance, I've used Sentence Transformers — which takes care of the proper normalization as recommended by the model creators. 

You've used the output from your own FastEmbed implementation — that's not the point of the test? How do you know that your implementation is correct — you don't. 

In fact, the Colab notebook I've shared proved that this implementation is wrong.